### PR TITLE
[FIX][15.0] pad: HTML is malformed when combining text type and Markup instance

### PR DESCRIPTION
**Pull request by royle-vietnam**
Description of the issue/feature this PR addresses:

Current behavior before PR:
Malformed HTML will receive a warning to the console log

Desired behavior after PR is merged:
Sets the text of a pad based on HTML, HTML must be well-formed

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by royleviindoo** [comment]
ping @seb-odoo @qdp-odoo @pedrobaeza

File: addons/pad/py_etherpad/__init__.py [hunk 1]
```diff
@@ -179,7 +181,7 @@ def setText(self, padID, text):
     def setHtmlFallbackText(self, padID, html):
         try:
             # Prevents malformed HTML errors
-            html_wellformed = '<html><body>' + html + '</body></html>'
+            html_wellformed = Markup("<html><body>") + html + Markup("</body></html>")
```
Comment (xmo-odoo) [inline addons/pad/py_etherpad/__init__.py:184]: Alternatively:
```suggestion
            html_wellformed = Markup("<html><body>%s</body></html>") % html
```
Bigger change but, I think, for the better.

**Review by xmo-odoo — APPROVED** [review]

File: addons/pad/py_etherpad/__init__.py (hunk 1 above)
Comment (royleviindoo) [reply addons/pad/py_etherpad/__init__.py]: @xmo-odoo thank you

**Comment by royleviindoo** [comment]
@xmo-odoo Can you call anyone who has permission to merge this PR?

**Review by odony — APPROVED** [review]
Thanks for the fix, this will work as expected as long as the `html` value is a proper Markup, which is supposed to be the case in all legit situations :+1: 

@robodoo override=ci/security r+

File: addons/pad/py_etherpad/__init__.py [hunk 2]
```diff
@@ -179,7 +181,7 @@ def setText(self, padID, text):
     def setHtmlFallbackText(self, padID, html):
         try:
             # Prevents malformed HTML errors
-            html_wellformed = '<html><body>' + html + '</body></html>'
+            html_wellformed = Markup("<html><body>%s</body></html>") % html
```
Comment (davidtranhp) [inline addons/pad/py_etherpad/__init__.py:184]: This should be
```python
            html_wellformed = Markup("<html><body>%s</body></html>" % html)
```

**Pull request closed without merging**